login-workflow/src/new-architecture/contexts/RegistrationContext/dictionaries/english.ts

```diff
@@ -35,9 +35,10 @@ const resources: RegistrationLanguageFile = {
             VERIFY_EMAIL: {
                 MESSAGE:
                     'A verification code has been sent to the email address you provided. Click the link or enter the code below to continue. This code is valid for 30 minutes.',
-                RESEND: 'Resend Verification Email',
+                RESEND: 'Send Again',
```

Comment:
No need to update this key. There is existing key shared directory `ACTIONS.RESEND`